### DISCUSSION TITLE
How to Backup the Apps I installed from Local?

### DISCUSSION THREAD

**Discussion by royzxje** [1Panel-dev/1Panel#4716] [Q&A]
Specifically, I installed ChatWoot App from Local, and I see that ChatWoot App is displayed in 1Panel's Container, but the problem here is that I cannot backup that app or I don't know how to do it.
Hope to receive feedback from you. 1Panel is truly amazing and I love it

**Comment by wanghe-fit2cloud** [comment]
Thank you for your feedback. 
Currently, we do not support backing up containers in the container list. 
However, you can consider packaging your application as a local app and then utilize the backup button provided in the installed app list to achieve the functionality mentioned above.


You can refer to the [official documentation](https://github.com/1Panel-dev/appstore/wiki/%E5%A6%82%E4%BD%95%E6%8F%90%E4%BA%A4%E8%87%AA%E5%B7%B1%E6%83%B3%E8%A6%81%E7%9A%84%E5%BA%94%E7%94%A8) or [forums for tutorials](https://bbs.fit2cloud.com/t/topic/640) on creating local applications.

**Comment by wanghe-fit2cloud** [comment]
![image](https://github.com/user-attachments/assets/85239778-9b8c-48da-bf0c-4f47d6cbe56e)
